plugins/quota/init.go

```diff
@@ -0,0 +1,80 @@
+package quota
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "quota"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "QuotaEnable",
+			Method:      "POST",
+			Pattern:     "/quota/{volname}",
+			Version:     1,
+			HandlerFunc: quotaEnableHandler},
+		route.Route{
+			Name:        "QuotaDisable",
+			Method:      "DELETE",
```

Comment:
This can be `POST` instead as no resource is being deleted. And the enable and disable APIs be `/quota/{volname}/enable` and ``/quota/{volname}/disable``

Reply:
with quota disable, the config file and a few xttars are deleted.Similarly with enable we create/set a few xttars. So Aravinda commented that its better to make use of the methods in the API for this instead of a separate API itself. and thus this change was done. And as a few things are removed I feel it makes sense to have it as DELETE method instead for the quota API instead of new API itself. The same goes for POST as enable. @aravindavk if you have any thoughts do let us know.

Reply:
Delete does delete all the limit that has been set previously.
the quota.conf (file / data from etcd) will be removed.